### ISSUE TITLE
Increase usability for python projects

### ISSUE THREAD

**Issue by lemmey** [lemmey/python_level_one#2]
word_count_main.py
data_generator_main.py

* add info screen
* user bugproof
* enhance interaction/feedback system

**Comment by Nandini1071** [comment]
Hi, is this issue still open ?

**Comment by lemmey** [comment]
> Hi, is this issue still open ?

Hello Nandini1071. The issue is still work in progress. Please feel free to have a look/share your ideas.

(Please note, this repository is more like a traning ground for the GitHub microcosm and might not remain open indefinitely.)

Best regards

**Comment by Nandini1071** [comment]
what enhancements you are looking ?

**Comment by Nandini1071** [comment]
or you want to add some more feature, if yes let me know what are all ?

**Comment by lemmey** [comment]
I simply thought about adding functionality to improve the 'user-experience'
* This could mean adding a menu with options the user can choose from
* * while making sure that user input is processed dynamically and correctly

PS I'am still learning how to do those things myself, but I'am happy for any suggestions. Cheers

**Comment by Nandini1071** [comment]
oh ok, so any thing on which you think I can contribute?

**Comment by lemmey** [comment]
In any case, in the the repository python_level_one there is a file named: data_cleaner_quick.py
I would like to enchance the program and make it more user-friendly.

If you want to wrap your head around it, you can try creating a #user menu for data_cleaner_quick.py,
from which the user can choose the pandas.method applied to the .csv file.
The key funcitons for now shoud involve:
`pd.drop_duplicates()`
`pd.dropna()`

The menu should revolve in a continuous #loop and the user should be able to exit by input of a keyword.

**Comment by Nandini1071** [comment]
I am not aware of pandas actually so can you tell me requirement for python itself.

**Comment by Nandini1071** [comment]
or else in this we can change the login in to python as we can use some inbuilt libraries, and also can add many features for this file, -> data_cleaner_quick.py

**Comment by Nandini1071** [comment]
I mean the features which you want I can implement in python itself in optimised way just explain the requirements.

**Comment by Nandini1071** [comment]
If you want with pandas then I can write with pandas also let me know your suggestions

**Comment by lemmey** [comment]
Nandini I appreciate your enthusiasm, note however that this is a student project - You can not make any mistakes here and I am happy to look at your own creative suggestions.

I am a beginner at progamming Python, so I try to learn about the workflow and project structure. 
Getting a second opinion from another developer would be of great value to me so I understand the GitHub workflow.

If you tell me a little bit about you maybe that helps finding a suitable project. 
The purpose of the projects in my repository are just for training and of course for anyone visiting the site.
Are you a student (beginner programmer) or do you just like to resolve open GitHub Issues?

Also, please excuse me if I cannot cater to your schedule

**Comment by Nandini1071** [comment]
I am good at python as intermediate you can consider me.

**Comment by Nandini1071** [comment]
so I can contribute to python and also for UI and backend projects, so I would like to contribute here.

**Comment by lemmey** [comment]
If you really want to collaborate on one of the projects, feel free to be proactive and bring in your own ideas, maybe we can make a polished application

**Comment by lemmey** [comment]
> so I can contribute to python and also for UI and backend projects, so I would like to contribute here.

Sounds terrific!

**Comment by Nandini1071** [comment]
And I am working professional as full stack developer.

**Comment by Nandini1071** [comment]
so what all ideas you are having I can contribute to all those projects.

**Comment by Nandini1071** [comment]
> How do you have time then to help little projects like this one :D ... no worries

No you said right this is for courses, so what kind of courses you are preparing and what all ideas you are having which is pending?

**Comment by lemmey** [comment]
Currently I'am just teaching myself. 

I will try to make up my mind on what needs to be done where, towards the end of the week. Please feel free to respond to any posted #Issues then.

**Comment by Nandini1071** [comment]
ok let me know if you have any issues with the code related in python would like to contribute.

**Comment by lemmey** [comment]
Thank you, I will try to think of something. Best regards

**Comment by Nandini1071** [comment]
yeah with python hell lot of projects are there which you can create.
